Description: add a minimal manpage for owexist
  Upstream manpage should probably be fully rewriten
--- /dev/null
+++ b/src/man/man1/owexist.1
@@ -0,0 +1 @@
+.so man1/owshell.1
\ No newline at end of file
--- a/src/man/man1/owshell.man
+++ b/src/man/man1/owshell.man
@@ -10,7 +10,7 @@
 '\"
 .TH OWSHELL 1 2004 "OWSHELL Manpage" "One-Wire File System"
 .SH NAME
-owdir, owread, owwrite, owget, owpresent \- lightweight owserver access
+owdir, owread, owwrite, owget, owexist, owpresent \- lightweight owserver access
 .SH SYNOPSIS
 .SS Minimal options
 .B owdir
--- a/src/man/man1/Makefile.am
+++ b/src/man/man1/Makefile.am
@@ -11,7 +11,7 @@
 ## man files that need no preprocessing
 dist_man1_MANS = \
     libowcapi.1 \
-    owdir.1 owget.1 owread.1 owwrite.1 owpresent.1 \
+    owdir.1 owget.1 owread.1 owwrite.1 owpresent.1 owexist.1 \
     ownetapi.1 ownetlib.1 libownet.1
 ## file to include in distribution
 EXTRA_DIST = $(SOFILES) $(MANFILES)
